### DISCUSSION TITLE
Download the Mojo compiler and have it work with integrated GPU?

### DISCUSSION THREAD

**Discussion by nav9** [modular/modular#144] [Q&A] [answered]
I'm going through the Mojo playground right now, and since it mentions the Mojo compiler and CUDA, I'd like to know a few things:

1. Will a Mojo compiler be available for download eventually, so that I'd be able to use it on my local system? If yes, how early do you estimate this would happen?
2. If it can be run locally, have you already implemented or do you plan to implement the functionality for the algorithms to make use of the integrated GPU on the processor if CUDA isn't available? I have a Ryzen 5 5600G, which has an integrated GPU on the processor itself, which is said to be as good as an NVidia GT 1030 graphics card. As of now, Keras/Tensorflow doesn't seem to be able to make use of this GPU because NVidia has monopolised CUDA. Could Mojo free us from this monopoly by allowing the use of any GPU?

**Comment by Mogball** [comment, marked as answer]
1. Yes, eventually.
2. That's the goal

**Comment by yoderj** [comment]
1. Now the Mojo compiler IS available for download, right?
2. Does it compile for CUDA?
